NHLBI_BioData_Catalyst/R/1_PICSURE_API_101.ipynb

```diff
@@ -368,7 +365,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "authQuery_categorical_example <- bdc::new.query(authPicSure)"
+    "query_categorical_example <- picsure::bdc.newQuery(session)"
    ]
   },
   {
```

Comment:
The following in-line documentation needs to be updated
| Method | Arguments / Input | Output|
|--------|-------------------|-------|
| query.select.add() | variable names (string) or list of strings | all variables included in the list (no record subsetting)|
| query.require.add() | variable names (string) or list of strings | all variables; only records that do not contain null values for input variables |
| query.anyof.add() | variable names (string) or list of strings | all variables; only records that contain at least one non-null value for input variables |
| query.filter.add() | variable name and additional filtering values | input variable; only records that match filter criteria |